test/parallel/test-stream-readable-collect.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+const common = require('../common');
+
+const { Readable } = require('stream');
+const assert = require('assert');
+
+const hello = Buffer.from('hello, ');
+const world = Buffer.from('world');
```

Comment:
Can you include multibyte characters in the test to make sure those are collected properly? (they are, but to make sure there are no regressions later)